### ISSUE TITLE
Provide resumableCurrentChunkSize on fileProgress

### ISSUE THREAD

**Issue by williamdes** [pointcloudtechnology/resumable.js#16]
For now I have made this code:

```js
    r.on('fileProgress', function (file, message, fileCategory) {
        var chunkSize = file.chunkSize;
        var total = file.file.size;
        var hash = file.uniqueIdentifier;
        var totalSent = parseInt($('#' + hash).data('total-sent') || 0, '10') + chunkSize;
        $('#' + hash).data('total-sent', totalSent);
        console.log(file, total, totalSent, totalSent / total * 100);
        var progress = parseInt(totalSent / total * 100, 10);
        if (progress === 100) {
            $('#pourcent').text('');
        }
        else {
            $('#pourcent').text(progress + '%');
        }
    });
```

This code is a bit wrong since the last chunk is not the size of a normal chunk, so it ends up like this: `117%`

It would be nice to have the `totalSent` or `currentChunkSize` to be able to compute a percentage of file sent.

**Comment by williamdes** [comment]
or should https://github.com/pointcloudtechnology/resumable.js/blob/0dc8a29358f693611cdc2ae81360ec5cd4514692/src/resumableFile.ts#L131

be fixed ?

In my events it stays at `_prevProgress: 0`

**Comment by UniquePanda** [comment]
Hi @williamdes 
If I see correctly you found the correct solution for your problem already. 😁 
As the "real" problem was a wrong explanation in the readme, this will be fixed with #9, which I will start working on now.

Hopefully once the readme is updated, we can more easily keep it up-to-date! 👍
